pkg/resources/metrics.go

```diff
@@ -246,6 +247,43 @@ func CreateRedisConnectivityAlert(ctx context.Context, client k8sclient.Client,
 	return pr, nil
 }
 
+
+func ReconcilePostgresFreeStorageAlerts(ctx context.Context, client k8sclient.Client, inst *v1alpha1.RHMI, cr *crov1.Postgres) error {
```

Comment:
any reason this could not be `CreatePostgresFreeStorageAlerts` to match the rest of the rules

Reply:
Because it is creating or updating the rule, so `reconciling` makes more sense IMO

Reply:
Would have said reconcilePrometheusRule does the create/update. nit picking never mind.